client/app/auth/service.js

```diff
@@ -0,0 +1,22 @@
+angular.module('throwcast.auth', [])
+
+.factory('authInterceptor', function ($q, $window){
+  return {
+    request: function (config) {
+      config.headers = config.headers || {};
+      if ($window.sessionStorage.token) {
+        config.headers.Authorization = 'Bearer ' + $window.sessionStorage.token;
+      }
+      return config;
+    },
+    response: function (rejection) {
+      if (response.status === 401) {
+        $log('Unauthorized User');
+      }
+      return $q.when(rejection);
+    }
+  };
+})
+.config(function ($httpProvider) {
```

Comment:
Missing a new line above this one if you want to be consistent with the extra lines you have between the module declaration and the chained calls.